bower.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
heh.. yeah I'd lose it except that it is our 1 non-RWell contribution: https://github.com/RoundingWellOS/marionette.toolkit/pull/101